spec/requests/groups_controller_spec.rb

```diff
@@ -90,4 +90,46 @@
       end
     end
   end
+
+  describe '#new' do
+    context 'when the user is not pek_admin' do
+      it 'returns forbidden' do
+        get '/groups/new'
+
+        expect(response).to have_http_status :forbidden
+      end
+    end
+
+    context 'when the user is pek_admin' do
+      let(:pek_admin) { create(:user, :pek_admin) }
+      before          { login_as(pek_admin) }
+      it 'renders the page' do
+        get '/groups/new'
+
+        expect(response).to have_http_status :ok
+        expect(response).to render_template :new
+      end
+    end
+  end
+
+  describe '#create' do
+    context 'when the user is not pek_admin' do
+      it 'returns forbidden' do
+        post '/groups'
+
+        expect(response).to have_http_status :forbidden
+      end
+    end
+
+    context 'when the user is pek_admin' do
+      let(:pek_admin) { create(:user, :pek_admin) }
+      before          { login_as(pek_admin) }
+      it 'creates the group' do
+        expect{
+          post '/groups', { group: FactoryBot.attributes_for(:group),
```

Comment:
I think here you can leave the `FactoryBot` as its helper is included.

Also please `headers:` and `params:` key to pass them instead of the positional args, because it's deprecated in Rails 5 and removed in 5.1.
```ruby
post '/groups', params: { group: FactoryBot.attributes_for(:group), ... }
```